package.json

```diff
@@ -69,23 +66,23 @@
     "vue-template-compiler": "2.6.14",
     "webpack": "5.74.0",
     "webpack-cli": "4.10.0",
-    "webpack-dev-middleware": "5.3.3",
-    "webpack-hot-middleware": "2.25.2",
+    "webpack-dev-server": "4.11.1",
     "webpack-merge": "5.8.0"
   },
   "scripts": {
     "clean": "rm -rf ./dist ./node_modules ./package-lock.json",
     "clean-test-lint": "npm run clean; npm install; npm run test; npm run lint",
-    "start": "node app.js",
+    "start": "npx webpack serve --config ./webpack.dev.js",
+    "start:coverage": "npx webpack serve --config ./webpack.coverage.js",
     "lint": "eslint example src e2e --ext .js,.vue openmct.js --max-warnings=0",
     "lint:fix": "eslint example src e2e --ext .js,.vue openmct.js --fix",
-    "build:prod": "cross-env webpack --config webpack.prod.js",
+    "build:prod": "webpack --config webpack.prod.js",
     "build:dev": "webpack --config webpack.dev.js",
     "build:coverage": "webpack --config webpack.coverage.js",
     "build:watch": "webpack --config webpack.dev.js --watch",
     "info": "npx envinfo --system --browsers --npmPackages --binaries --languages --markdown",
-    "test": "cross-env NODE_ENV=test NODE_OPTIONS=\"--max_old_space_size=4096\" karma start --single-run",
```

Comment:
node memory limits increased to 4GB with node 14 so we no longer need to set this at runtime